qdrant/v1.2.x/points.md

```diff
@@ -313,10 +313,8 @@ see [update vectors](#update-vectors).
 
 ## Modify points
 
-You can modify a point in two ways. The first is to modify the vector.
-Currently, you would need to re-upload the point to modify the vector.
-
-The second is to modify the payload, for which there are several methods.
+You can modify a point in two ways. You can either modify the vectors or modify
+the payload. For both are several methods.
```

Comment:
"There are two ways to do this." ......or....."There are several ways to do this."

Reply:
This tries to say:
- there are two ways to modify a point (modify its vectors, or modify its payload)
- there are several methods to modify these vectors and payloads

What would be a good replacement sentence? What about this?

> To change a point, you can modify its vectors or its payload. There are several ways to do this.

Reply:
I like your last suggestion.

> To change a point, you can modify its vectors or its payload. There are several ways to do this.

My brain had trouble parsing "For both are several methods".